doc/changelog.rst

```diff
@@ -14,6 +14,7 @@ Core
 
 - Move calculate_taxes_automatically from ``OrderSource`` to ``TaxModule``
 - Add shop product validation for OrderSource
+- Add order_creator_finished signal ``under order_creator``
```

Comment:
The _under_ should be outside the double backquotes. (And `order_creator_finished` should be in double backquotes too, as it's an identifier.)